csharp/ql/lib/semmle/code/csharp/Member.qll

```diff
@@ -222,33 +259,54 @@ class Modifiable extends Declaration, @modifiable {
 }
 
 /** A declaration that is a member of a type. */
-class Member extends DotNet::Member, Modifiable, @member {
+class Member extends Modifiable, @member {
   /** Gets an access to this member. */
   MemberAccess getAnAccess() { result.getTarget() = this }
 
+  /** Holds if this member is declared `public`. */
```

Comment:
Can all these overrides be removed now? At least we don't need to add the QL doc.